File: CS/DXDiagram.CustomDiagramStorage/DiagramStorage.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DXDiagram.CustomDiagramStorage {
    public class DiagramStorage : DbContext {
        public DbSet<DiagramData> DiagramData { get; set; }
    }
    public class DiagramData {
        public int Id { get; set; }
        public string Name { get; set; }
        public byte[] Data { get; set; }
    }
}

File: VB/DXDiagram.CustomDiagramStorage/DiagramStorage.vb
Imports System
Imports System.Collections.Generic
Imports System.Data.Entity
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DXDiagram.CustomDiagramStorage
    Public Class DiagramStorage
        Inherits DbContext

        Public Property DiagramData() As DbSet(Of DiagramData)
    End Class
    Public Class DiagramData
        Public Property Id() As Integer
        Public Property Name() As String
        Public Property Data() As Byte()
    End Class
End Namespace
